net/adguardhome/Makefile

```diff
@@ -6,13 +6,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=adguardhome
-PKG_VERSION:=0.107.36
+PKG_VERSION:=0.107.41
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
```

Comment:
line duplication? Otherwise thanks for the update!

Reply:
I have no idea how this happened xd